web/src/components/layout/logo.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 import { LocaleLink } from '../locale-helpers';
+import * as logo from '../../../img/cv-logo-one-color-white.svg';
```

Comment:
Oh, so the `~img/cv-logo-one-color-white.svg` wouldn't work here?

Reply:
Oh, may be now it is done